<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="HomePage"
           tests="6"
           failures="1"
           errors="0"
           skipped="0"
           time="0.501">
  <properties />
            <testcase name="testAvailableColorIsDisplayedOnNextTileWhenAMeetingIsCurrentInTheRoomAndIsFreeEOD" time="0.039" classname="tests.tablet.homeUI.CurrentMeetingValuesVerifications">
                <failure type="java.lang.AssertionError"
                  message="expected [null] but found [rgba(0, 128, 0, 1)]"
              >
<![CDATA[java.lang.AssertionError: expected [null] but found [rgba(0, 128, 0, 1)]
  org.testng.Assert.fail(Assert.java:94)
  org.testng.Assert.failNotEquals(Assert.java:494)
  org.testng.Assert.assertEquals(Assert.java:123)
  org.testng.Assert.assertEquals(Assert.java:176)
  org.testng.Assert.assertEquals(Assert.java:186)
  tests.tablet.homeUI.CurrentMeetingValuesVerifications.testAvailableColorIsDisplayedOnNextTileWhenAMeetingIsCurrentInTheRoomAndIsFreeEOD(Unknown Source)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
  java.lang.reflect.Method.invoke(Unknown Source)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
  org.testng.TestRunner.privateRun(TestRunner.java:767)
  org.testng.TestRunner.run(TestRunner.java:617)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
  org.testng.SuiteRunner.run(SuiteRunner.java:240)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
  org.testng.TestNG.run(TestNG.java:1057)
  org.testng.TestNG.privateMain(TestNG.java:1364)
  org.testng.TestNG.main(TestNG.java:1333)
]]>
      </failure>

          </testcase>
  
  
            <testcase name="testCurrentMeetingOrganizerIsDisplayedOnNowTile" time="0.053" classname="tests.tablet.homeUI.CurrentMeetingValuesVerifications" />
                <testcase name="testCurrentMeetingSubjectDisplayedOnNowTile" time="0.130" classname="tests.tablet.homeUI.CurrentMeetingValuesVerifications" />
                <testcase name="testAvailableMessageDisplayedInNextTileWhenRoomIsFreeAfterAMeeting" time="0.141" classname="tests.tablet.homeUI.CurrentMeetingValuesVerifications" />
                <testcase name="testBussyColorIsDisplayedOnNowTileWhenAMeetingIsInProgressInTheRoom" time="0.031" classname="tests.tablet.homeUI.CurrentMeetingValuesVerifications" />
                <testcase name="testIntervalTimeDisplayedForNextMeetingInRoomIsFromEndTimeCurrentMeetingToMidnight" time="0.107" classname="tests.tablet.homeUI.CurrentMeetingValuesVerifications" />
      
</testsuite>
